lib/screens/profile_page.dart

```diff
@@ -36,31 +36,34 @@ class _ProfilePageState extends State<ProfilePage> {
         trailing: PreferanceButton(),
         border: null,
       ),
-      child: SafeArea(
-        minimum: EdgeInsets.symmetric(horizontal: 16),
-        child: Column(
-          crossAxisAlignment: CrossAxisAlignment.start,
-          children: [
-            SizedBox(height: 16),
-            Text(
-              'Greetings',
-              style: Style.largeTitle(context),
-            ),
-            SizedBox(height: 16),
-            Text(
-              user.name,
-              style: Style.title(context),
-            ),
-            SizedBox(height: 32),
-            Stats(),
-            SizedBox(height: 32),
-            Text(
-              'Analytics',
-              style: Style.title2(context),
-            ),
-            SizedBox(height: 16),
-            Center(child: Analytics())
-          ],
+      child: SingleChildScrollView(
+        scrollDirection: Axis.vertical,
```

Comment:
Scroll axis is vertical by default. you do not need to explicitly write that.
You can remove it.
https://api.flutter.dev/flutter/widgets/SingleChildScrollView/scrollDirection.html

Reply:
Updated